teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -209,14 +216,22 @@ The network interface(s) on which the node listens for P2P communication.
       hidden = true)
   private Integer minimumRandomlySelectedPeerCount;
 
+  @Option(
+      names = {"--p2p-static-peers-url"},
+      paramLabel = "<URL>",
+      description =
+          "Specifies a URL or file containing a list of 'static' peers (one per line) with which to establish and maintain connections",
+      arity = "1")
+  private String p2pStaticPeersUrl;
+
   @Option(
       names = {"--p2p-static-peers"},
       paramLabel = "<PEER_ADDRESSES>",
       description =
-          "Specifies a list of 'static' peers with which to establish and maintain connections",
+          "Specifies a comma-separated list of 'static' peers with which to establish and maintain connections",
       split = ",",
-      arity = "0..*")
-  private List<String> p2pStaticPeers = new ArrayList<>();
+      arity = "1")
```

Comment:
note to self, need to make sure the old peers cli works the same way still, as that functionality definitely used

Reply:
this is a note for me, thats what i meant by 'note to self' sorry :) I'll be looking at this PR today.